tests/test_models/test_mutables/test_diffop.py

```diff
@@ -107,11 +109,17 @@ def test_forward(self):
         input = torch.randn(4, 32, 64, 64)
 
         # test set_forward_args
-        arch_param = op.build_arch_param()
+        diffmutator = DiffModuleMutator()
```

Comment:
Use user-defined instead  of `mutator`  to construct `arch_param`, as we hope the mutable could work(build and forward) independently in our design.